Support more bond options (#452)
linux-system-roles/network: README.md

    `all_slaves_active` in kernel and NetworkManager. The boolean value `False` drops
    the duplicate frames (received on inactive ports) and the boolean value `True`
    delivers the duplicate frames.

- `arp_all_targets`

    This option specifies the quantity of arp_ip_targets that must be reachable in
    order for the ARP monitor to consider a port as being up. The possible values are
    `any` or `all`.

- `arp_interval`

    This option specifies the ARP link monitoring frequency in milliseconds. A value of
    0 disables ARP monitoring.

- `arp_validate`

    In any mode that supports arp monitoring, this option specifies whether or not ARP
    probes and replies should be validated. Or for link monitoring purposes, whether
    non-ARP traffic should be filtered (disregarded). The possible values are: `none`,
    `active`, `backup`, `all`, `filter`, `filter_active`, `filter_backup`.

- `arp_ip_target`

    When `arp_interval` is enabled, this option specifies the IP addresses to use as
    ARP monitoring peers.

- `downdelay`

    The time to wait (in milliseconds) before disabling a port after a link failure
    has been detected.

- `fail_over_mac`

>     This option specifies whether active-backup mode should set all ports to the same
>     MAC address at enslavement (the traditional behavior), or, when enabled, perform
>     special handling of the bond's MAC address in accordance with the selected policy.
>     The possible values are: `none`, `active`, `follow`.

Reviewer: ```suggestion
    This option specifies the policy to select the MAC address for the bond interface in active-backup mode. 
    The possible values are: `none` (default), `active`, `follow`.
```

Author: Thanks. I updated again.

- `lacp_rate`

    In `802.3ad` bonding mode, this option defines the rate in which we requst link
    partner to transmit LACPDU packets. The possible values are: `slow`, `fast`.

- `lp_interval`

    This option specifies the number of seconds between instances where the bonding
    driver sends learning packets to each slaves peer switch.

- `miimon`

    Sets the MII link monitoring interval (in milliseconds).

- `min_links`

    This option specifies the minimum number of links that must be active before
    asserting the carrier.

- `num_grat_arp`

    This option specify the number of peer notifications (gratuitious ARPs) to be
    issued after a failover event. The allowed range for the value is 0 - 255.

- `packets_per_port`

    In `balance-rr` bonding mode, this option specifies the number of packets allowed
    for a slave in network transmission before moving to the next one. The allowed
    range for the value is 0 - 65535.

- `peer_notif_delay`

    This option specifies the delay (in milliseconds) between each peer notification
    when they are issued after a failover event.

- `primary`

>     This option defines which slave is the primary device.

Reviewer: ```suggestion
    This option defines the primary port.
```

Author: I updated. Thanks.
